packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -0,0 +1,348 @@
+            // "WETH"
+            hex"5745544800000000000000000000000000000000000000000000000000000008"
+        );
+        vm.store(
+            Predeploys.WETH9,
+            // uint8 public decimals
+            hex"0000000000000000000000000000000000000000000000000000000000000002",
+            // 18
+            hex"0000000000000000000000000000000000000000000000000000000000000012"
+        );
+    }
+
+    // TODO Not sure how to set `StandardBridge public otherBridge;`
+    // L2StandardBridge's constructor doesn't take args, but calls
+    // `initialize({ _otherBridge: StandardBridge(payable(address(0))) });`
+    // which permanetly sets `StandardBridge public otherBridge`
+    function _setL2StandardBridge() internal {
+        L2StandardBridge bridge = new L2StandardBridge();
+        address impl = _predeployToCodeNamespace(Predeploys.L2_STANDARD_BRIDGE);
+
+        vm.etch(impl, address(bridge).code);
+        vm.store(
+            Predeploys.L2_STANDARD_BRIDGE,
+            // uint8 private _initialized;
+            // bool private _initializing;
+            hex"0000000000000000000000000000000000000000000000000000000000000000",
+            // 1
+            // false
+            hex"0000000000000000000000000000000000000000000000000000000000000001"
+        );
+        vm.store(
+            Predeploys.L2_STANDARD_BRIDGE,
+            // CrossDomainMessenger public messenger;
+            hex"0000000000000000000000000000000000000000000000000000000000000003",
+            // Predeploys.L2_CROSS_DOMAIN_MESSENGER
+            hex"0000000000000000000000004200000000000000000000000000000000000007"
+        );
+        vm.store(
+            Predeploys.L2_STANDARD_BRIDGE,
+            // StandardBridge public otherBridge;
+            hex"0000000000000000000000000000000000000000000000000000000000000004",
+            // L1StandardBridgeProxy
+            hex"0000000000000000000000000c8b5822b6e02cda722174f19a1439a7495a3fa6"
+        );
+
+        vm.etch(address(bridge), hex"");
+        vm.resetNonce(address(bridge));
+    }
+
+    function _setL2CrossDomainMessenger() internal {
+        L2CrossDomainMessenger messenger = new L2CrossDomainMessenger();
+        address impl = _predeployToCodeNamespace(Predeploys.L2_CROSS_DOMAIN_MESSENGER);
+
+        vm.etch(impl, address(messenger).code);
+
+        vm.store(
+            Predeploys.L2_CROSS_DOMAIN_MESSENGER,
+            // TODO - I'm expecting these to be the values for slot 0, but
+            // the value of slot doesn't seem to match the below expectation.
+            // I think there may be another variable sharing the slot
+            // uint8 private _initialized;
+            // bool private _initializing;
+            hex"0000000000000000000000000000000000000000000000000000000000000000",
+            // 1
+            // false
+            hex"0000000000000000000000010000000000000000000000000000000000000000"
```

Comment:
I'm unsure of what other variables are being set in this storage slot, was expecting to match the other contracts and look like:
```solidity
vm.store(
            Predeploys.L2_STANDARD_BRIDGE,
            // uint8 private _initialized;
            // bool private _initializing;
            hex"0000000000000000000000000000000000000000000000000000000000000000",
            // 1
            // false
            hex"0000000000000000000000000000000000000000000000000000000000000001"
        );
```